tests/cms/views/push_notifications/test_push_notification_form_view.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from django.conf import settings
+from django.test.client import Client
+from django.urls import reverse
+
+from tests.conftest import (
+    ANONYMOUS,
+    PRIV_STAFF_ROLES,
+    STAFF_ROLES,
+)
+
+
+@pytest.mark.django_db
+def test_validate_forms(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+) -> None:
+    client, role = login_role_user
+
+    edit_push_notification = reverse(
+        "edit_push_notification",
+        kwargs={
+            "push_notification_id": 1,
+            "region_slug": "augsburg",
+            "language_slug": "de",
+        },
+    )
+    response = client.post(
+        edit_push_notification,
+        data={
+            "translations-TOTAL_FORMS": "2",
+            "translations-INITIAL_FORMS": "0",
+            "translations-MIN_NUM_FORMS": "1",
+            "translations-MAX_NUM_FORMS": "2",
+            "translations-0-language": "1",
+            "translations-0-title": "",
+            "translations-0-text": "Test content",
+            "translations-1-language": "2",
+            "translations-1-title": "",
+            "translations-1-text": "",
+            "regions": [1],
```

Comment:
Can we probably add another region whose default language is not German and test the title field is reall required in both German and the language of another region? In this way the test can cover the case in which the bug was originally reported 💪

Reply:
@MizukiTemma I have added English as a default language and added another test for that. I am not sure whether it is a right way to test it :thinking:

Reply:
Sorry for my bad explanation 🙈 The case I meant is the following one:

A region with German as default language (e.g. Augsburg) creates a push notification only with German translation but checked another region with English as default language (e.g. Berlin) too in the settings for "Regions". In this case the warning "Title (English): This field is required." must be shown, because there is no English translation while it is needed for Berlin. It must go well but after creating an English translation, because then both Augsburg and Berlin are happy with their default language covered.